src/Sarif.Viewer.VisualStudio/ErrorList/SarifResultTableEntry.cs

```diff
@@ -182,19 +183,20 @@ private void ErrorListInlineLink_Click(object sender, RoutedEventArgs e)
 
                 if (location != null)
                 {
-                    // Set the current SARIF error in the manager so we track code locations.
-                    CodeAnalysisResultManager.Instance.CurrentSarifResult = this.Error;
-
-                    SarifViewerPackage.SarifToolWindow.Control.DataContext = null;
-
                     if (this.Error.HasDetails)
                     {
-                        // Setting the DataContext to null (above) forces the TabControl to select the appropriate tab.
-                        SarifViewerPackage.SarifToolWindow.Control.DataContext = this.Error;
+                        IComponentModel componentModel = (IComponentModel)Package.GetGlobalService(typeof(SComponentModel));
+                        if (componentModel != null)
+                        {
+                            ISarifErrorListEventSelectionService sarifSelectionService = componentModel.GetService<ISarifErrorListEventSelectionService>();
+                            if (sarifSelectionService != null)
+                            {
+                                sarifSelectionService.NavigatedItem = this.Error;
```

Comment:
> sarifSelectionService.NavigatedItem = this.Error; [](start = 31, length = 50)

Comment that this will ultimately change what is being displayed in the explorer window, while the navigate below will move the editor to the "location"